exporters/stats/stackdriver/README.md

```diff
@@ -94,5 +95,11 @@ StackdriverStatsExporter.createAndRegisterWithProjectId("MyStackdriverProjectId"
 Java 7 or above is required for using this exporter.
 
 ## FAQ
+### Why did I get a PERMISSION_DENIED error from Stackdriver when using this exporter?
+To use our Stackdriver Stats exporter, your Stackdriver account needs to have permission to [create
+custom metrics](https://cloud.google.com/monitoring/custom-metrics/creating-metrics), and that 
+requires a [Premium tier Stackdriver account](https://cloud.google.com/monitoring/accounts/tiers#this_request_is_only_available_in_the_premium_tier). 
+Please note that by default all new Stackdrvier accounts are Basic tier. To upgrade to a Premium 
```

Comment:
Typo in stackdriver

Reply:
Good catch! Post https://github.com/census-instrumentation/opencensus-java/pull/835 for this.